lib/grpcx/server/interceptors/instrumentation.rb

```diff
@@ -1,8 +1,12 @@
 require 'active_support/notifications'
+require 'grpclb/server'
 
 module Grpcx
-  module Server
+  class Server < Grpclb::Server
```

Comment:
I dislike this, but seems no way round it, except, probably, changing naming scheme to smth like `Grpcx::Interceptors::Server::Instrumentation`